packages/types/src/swap/pool.ts

```diff
@@ -12,18 +12,8 @@ export type SwapPoolProvider =
   | 'vyfi'
   | 'spectrum'
 
-export type SwapSupportedProvider = Extract<
-  SwapPoolProvider,
-  | 'minswap'
-  | 'wingriders'
-  | 'sundaeswap'
-  | 'muesliswap'
-  | 'muesliswap_v2'
-  | 'vyfi'
->
-
 export type SwapPool = {
-  provider: SwapSupportedProvider
+  provider: SwapPoolProvider
```

Comment:
eventually will become a string, otherwise the parser will throw if a new is added on the fly.